dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo

```diff
@@ -17,16 +17,25 @@ model GeneratorPV "Model for generator PV based on SignalN for the frequency han
   extends BaseClasses.BasePV;
   extends BaseClasses.BaseQStator(QStatorPu(start = QGen0Pu * SystemBase.SnRef / QNomAlt));
 
-equation
-  when QGenPu <= QMinPu and UPu >= URefPu then
+equatio
+  when QGenPu + QDeadBandPu <= QMinPu and UPu - UDeadBandPu >= URefPu then
```

Comment:
Is it better to remove equality here? (it is still added in some models) I am not sure if it impacts the results
```suggestion
  when QGenPu + QDeadBandPu <= QMinPu and UPu - UDeadBandPu > URefPu then
```

Reply:
Done